### ISSUE TITLE
[hugo-0.18+] "Everything is a Page"

### ISSUE THREAD

**Issue by adrianovieira** [humboldtux/startbootstrap-clean-blog#28]
I've built a hotsite using your theme and I use *Hugo-0.16" to render it.

Now I'm trying to update to *Hugo-0.18+*, but I'm having some problem. I believe the troubles are related to the "rule" *everything as a page*.

According to the new rules of *Hugo-0.18+* threat *everything as a page* (https://gohugo.io/content/using-index-md/) if we use ***_index.md*** and ‘Everything is a Page’.

Then I created a sample site to test, but it doesn't work.

```
.
├── archetypes
├── config.toml
├── content
│   ├── about
│   │   └── _index.md
│   └── _index.md
├── data
├── layouts
├── static
└── themes
    └── startbootstrap-clean-blog
        ├── ...
```

May be it isn't an issue in your template itself, but hope you can point me some solution or workaround. Could you?

**Comment by humboldtux** [comment]
I'm sorry @adrianovieira but i don't understand which problem in need to fix: can you elaborate on "I created a sample test site to check, but it doesn't work"? What went wrong?

**Comment by adrianovieira** [comment]
Ok! Let me try.

I've created a sample (https://github.com/adrianovieira/poc-hugo-issue-3387) and I expected it should work with your theme and with Hugo-0.18+.

I don't know what's wrong there.

**Comment by humboldtux** [comment]
What error message did you get?

**Comment by adrianovieira** [comment]
We also have a hotsite (http://gitlab.com/devops-df/2017.brasilia.devopsdays.com.br) that works with Hugo-0.17 (and Hugo-0.16).

But if we update your theme on that hotsite and try to use hugo-0.18+ some pages (e.g "about") doesn't work properly.

**Comment by adrianovieira** [comment]
For all cases, the page content isn't shown.

**Comment by humboldtux** [comment]
Are you still experiencing this issue?

**Comment by adrianovieira** [comment]
Yes! But we will review our wesite and maybe we will migrate it to a CMS. 

Thanks by your great job in this theme and also by your attention.
